README.md

```diff
@@ -79,6 +79,7 @@ This list started from a discussion in DataTalks.Club. Want to talk about data?
 
 ## Data Enginering, Big Data, etc
 
+- [Weaviate vector search podcast](https://weaviate.io/podcast.html)
```

Comment:
Can you please add it to the end of the list?

Reply:
I mean after the airflow pod

Reply:
Ok - it's now _after_ Airflow. Please let me know if this is ok.